packages/hash/frontend/src/blocks/page/Comments/style.module.css

```diff
@@ -0,0 +1,5 @@
+.Prosemirror_Input {
+  width: 100%;
+  outline: none;
+  padding: 13.5px 0;
```

Comment:
I went with this padding so that the total height of the input is 48px (13.5px padding top + 21px height + 13.5px padding bottom). Making this change would leave it at 49px. Do you still think we should make this change?

Reply:
This will leave the whole element offset by half a pixel which could cause some funky aliasing. Have you considered making the height even instead?

Reply:
I can't set the height to 20px because the input is multi-line and that would prevent it from expanding. Do we care more about the aliasing problem or about it being exactly 48px?